stock_cycle_count/models/stock_inventory.py

```diff
@@ -15,7 +15,10 @@ class StockInventory(models.Model):
     @api.multi
     @api.depends("state", "line_ids")
     def _compute_inventory_accuracy(self):
-        for inv in self:
+        zero_inventories = self.filtered(
+            lambda i: not i.line_ids and i.state == 'done')
+        zero_inventories.inventory_accuracy = PERCENT
```

Comment:
@rousseldenis This line of course give singleton error, but the obivous solution (`zero_inventories.write({'inventory_accuracy': PERCENT})`) is creating an infinite loop, I really don't understand why. Any idea?